src/index.ts

```diff
@@ -1,5 +1,5 @@
 export * from './enums';
 export * from './types';
-import IVSPlayer from './IVSPlayer';
+export { default as IVSPlayer } from './IVSPlayer';
+export type { IVSPlayerProps } from './IVSPlayer';
```

Comment:
Since the component we export is the `IVSPlayerContainer`
```
const IVSPlayerContainer = React.forwardRef<IVSPlayerRef, Props>
```
The correct type to export here is `type Props = {` 

However can we split changes to how we export types / components to a separate PR?

Reply:
Thank you for the feedback, I've moved these changes over to [this](https://github.com/aws/amazon-ivs-react-native-player/pull/152) PR